src/context.h

```diff
@@ -251,9 +251,9 @@ class Context {
 
   /// Adds a package type to a simulation-wide accessible list.
   /// Agents should NOT add their own packages.
-  void AddPackage(std::string name, double fill_min = 0,
-                  double fill_max = std::numeric_limits<double>::max(),
-                  std::string strategy = "first");
+  Package::Ptr AddPackage(std::string name, double fill_min = 0,
```

Comment:
Is there a reason that this returns a Ptr to the package now? I don't see it used anywhere. Not a reason to block, but curious.

Reply:
Nothing uses this anymore except in testing (which I'll change)